frigate/config.py

```diff
@@ -816,109 +817,113 @@ def runtime_config(self) -> FrigateConfig:
             exclude_unset=True,
         )
 
-        for name, camera in config.cameras.items():
+        for name, camera in config.cameras.copy().items():
```

Comment:
Why was copy added?

Reply:
Nevermind I get it. I think this would be easier to just add a filter instead of adding if / else.

Reply:
If you use it without, you'll get an error when modifying a lilst while iterating it. With the "copy trick" this can be avoided.

Reply:
See my second comment. Another option besides using `filter` to avoid the large if / else could be doing 

```
if not camera.enabled:
  config.cameras.pop(camera)
  continue
```

Reply:
When I switch to this option I get an error
"TypeError: unhashable type: CameraConfig"
Just trying to understand why...

Reply:
Which option (filter or the code snippet I put)? 

To be clear you'll still need the copy() with the code snippet I put above, it'll help clean up the large if / else that we'd prefer to avoid